playwright/__init__.py

```diff
@@ -17,3 +17,4 @@
 Firefox and WebKit with a single API. Playwright is built to enable cross-browser
 web automation that is ever-green, capable, reliable and fast.
 """
+__version__ = "1.14.0"
```

Comment:
we don't want to maintain another version, you can get the version from `from playwright._repo_version import version`